playground/mocks/config/responseConfig.js

```diff
@@ -105,6 +105,7 @@ const idx = {
   ],
   '/idp/idx/enroll': [
     'enroll-profile-new',
+    // 'enroll-profile-with-password',
```

Comment:
nit: is this intended? Can you consider removing the commented out code?

Reply:
Yes, this was intended - to allow for users to visualize what it's like to have the password prompt on the first page of SSR, waiting for a review from the SIW team if we can keep it in